easybuild/easyconfigs/t/Trinity/Trinity-2.0.6-goolf-1.4.10.eb

```diff
@@ -0,0 +1,43 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'MakeCp'
```

Comment:
use available easyblock for trinity instead, so everything gets built as it should be?